### PR TITLE
Document Configuration and MediaEngine re-use

### PR DESCRIPTION
**Pull request by josharian**
I did a quick pass through to check that these claims are true (and I want them to be true!), but I am new to this codebase, so I could be wrong. Please review skeptically.

### CODE REVIEW

File: mediaengine.go [hunk 1]
```diff
@@ -26,34 +26,40 @@ const (
...
 	m.RegisterCodec(NewRTPVP8Codec(DefaultPayloadTypeVP8, 90000))
 	m.RegisterCodec(NewRTPVP9Codec(DefaultPayloadTypeVP9, 90000))
 	m.RegisterCodec(NewRTPH264Codec(DefaultPayloadTypeH264, 90000))
 }
 
-// PopulateFromSDP finds all codecs in a session description and adds them to a MediaEngine, using dynamic
-// payload types and parameters from the sdp.
+// PopulateFromSDP finds all codecs in sd and adds them to m, using the dynamic
+// payload types and parameters from sd.
+// PopulateFromSDP is not safe for concurrent use.
```
Comment (Sean-Der) [inline mediaengine.go:62]: Not sure how to word this properly, but `PopulateFromSDP` is meant to be used when you are answering. The offerer controls the PayloadTypes for the call, and this allows an answerer to match the PayloadTypes properly of the offerer.

You should also probably only call `PopulateFromSDP` once on a MediaEngine, and then use it only for that single session.

Comment (josharian) [reply mediaengine.go]: Thanks. I took a stab and wordsmithing. Tell me what you think. It's slightly verbose, but I think it's better to be a bit too verbose than a bit too terse when it comes to docs.

**Comment by josharian** [comment]
Just like https://github.com/pion/sdp/pull/52, Travis is complaining that:

> it failed 'Separate subject from body with a blank line'

But I don't see any commit messages that match that description. I really want to want to contribute to pion, but the constant battles with the linter are pretty frustrating and time consuming.

**Comment by Sean-Der** [comment]
Merged!

The issue was that `Fix broken link to issue in comment` didn't have a body, just added one and we are all good :)

**Pull request closed without merging**